<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AERPEditUserAccessRow</class>
 <widget class="QDialog" name="AERPEditUserAccessRow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edición de permisos de acceso por registro [*]</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/aplicacion/images/BolaAleph.png</normaloff>:/aplicacion/images/BolaAleph.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Seleccione Roles o Usuarios</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Roles</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Usuarios</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QListWidget" name="lwRoles"/>
      </item>
      <item row="1" column="1">
       <widget class="QListWidget" name="lwUsers"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Seleccione los permisos</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="chkRead">
        <property name="whatsThis">
         <string>Permite acceder en modo lectura a los datos del usuario</string>
        </property>
        <property name="text">
         <string>Lectura o Visualización</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkEdit">
        <property name="whatsThis">
         <string>Permite la modificación y/o borrado de los datos del registro y del propio registro</string>
        </property>
        <property name="text">
         <string>Edición o modificación y borrado</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkSelectable">
        <property name="whatsThis">
         <string>Permite seleccionar este registro desde otros que puedan referenciarlo</string>
        </property>
        <property name="text">
         <string>Seleccionable</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pbOk">
     <property name="text">
      <string>&amp;Ok</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/aplicacion/images/ok.png</normaloff>:/aplicacion/images/ok.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pbCancel">
     <property name="text">
      <string>&amp;Cancelar</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/generales/images/close.png</normaloff>:/generales/images/close.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lwRoles</tabstop>
  <tabstop>lwUsers</tabstop>
  <tabstop>chkRead</tabstop>
  <tabstop>chkEdit</tabstop>
  <tabstop>pbOk</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
